frontend/src/components/Button.tsx

```diff
@@ -31,11 +34,20 @@ type Props = {
   disabled?: boolean
   type?: 'button' | 'submit' | 'reset' | undefined
   onClick: () => void
+  setWidth?: string | undefined
+  size?: 'small' | 'regular' | undefined
 }
 
 function ButtonComponent(prop: Props) {
   return (
-    <Button onClick={() => prop.onClick()} state={prop.state} disabled={prop.disabled} type={prop.type ?? 'button'}>
+    <Button
+      onClick={() => prop.onClick()}
+      state={prop.state}
+      disabled={prop.disabled}
+      type={prop.type ?? 'button'}
+      width={prop.setWidth ?? 'max-content'}
```

Comment:
consider doing this in the styled component instead

`width: ${(props) => props.width ?? 'max-content'};`

Reply:
fixed in 4c4466139